cli/src/main/java/org/jboss/as/cli/handlers/OperationRequestHandler.java

```diff
@@ -184,18 +184,13 @@ private ModelNode validateRequest(CommandContext ctx, ModelNode request) throws
                     throw new CommandFormatException("Operation '" + operationName + "' does not expect any property.");
                 }
             }
```

Comment:
BTW, if current master L177 is reverted back to what it was before to fix my comment there, then the block whose beginning isn't shown in this diff but which ends here is pointless. That is, it's pointless in the current code. It just repeats an if test that has already evaluated false at L177.